tests/puppeteer/.eslintrc.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
     sourceType: 'module',
   },
   rules: {
-    'no-plusplus': [2, { allowForLoopAfterthoughts: true }]
+    'no-plusplus': [2, { allowForLoopAfterthoughts: true }],
+    "max-len": [2, {"code": 120}]
```

Comment:
```suggestion
    'max-len': [2, {code: 120}]
```

Reply:
Eslint don't detect errors in eslint config file.
didn't see it, thank you